projects/fsastorefrontlib/src/core/my-account/services/policy-chart-data.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@angular/core';
+import { LanguageService } from '@spartacus/core';
+import { EChartsOption } from 'echarts';
+import { tap } from 'rxjs/operators';
+
+@Injectable()
+export class PolicyChartDataService {
+  language;
+
+  constructor(protected languageService: LanguageService) {
+    this.languageService
```

Comment:
Is language service really needed here? In any case we should avoid subscribing in services unless absolutely necessary.

Reply:
Maybe I can remove `languageService` in the component, do subscribing there and pass language as parameter to `calculatePremiumAmountByCategory` method?